hat/app/org/hatdex/hat/utils/Mailer.scala

```diff
@@ -206,4 +214,21 @@ class HatMailerImpl @Inject() (
     )
     Done
   }
+
+  // change txt and html to
+  // hatClaimed.scala.{txt|html}
+  // data is ok
+  def hatClaimed(email: String,
+                 loginLink: String
+    )(implicit messages: MessagesApi,
+                   lang: Lang,
+                   server: HatServer): Done = {
+    sendEmail(email)(
+      from = emailFrom,
+      subject = messages("email.dataswift.auth.subject.verifyEmail"),
+      bodyHtml = views.html.mails.emailHatClaim(email, loginLink),
```

Comment:
The code can be made more readable by changing template name to align with current vocabulary. Term "claim" is being faded out.